### PR TITLE
RUC LSM update

### LINKED ISSUE #117: RUC LSM - adding leaf area index and other surface parameters to the history
## Description
Provide a clear and concise description of the problem to be solved.
Missing output that is requested by the stakeholders.
## Solution
Add a clear and concise description of the proposed solution.
The missing variables are added to the history files.
## Alternatives (optional)
If applicable, add a description of any alternative solutions or features you've considered.

## Related to (optional)
Directly reference any issues or PRs in this or other repositories that this is related to, and describe how they are related.
PR#116

### PR DESCRIPTION
**Pull request by tanyasmirnova**
## PR Author Checklist:

- [x] I have linked PR's from all sub-components involved in section below. 
- [ ] I am confirming reviews are completed in ALL sub-component PR's.
- [x] I have run the full RT suite on either Hera/Cheyenne AND have attached the log to this PR below this line:
  - LOG: 
- [ ] I have added the list of all failed regression tests to "Anticipated changes" section.
- [ ] I have filled out all sections of the template.

## Description

1. Adding leaf area index, wilting point and soil field capacity to the history files.
2. Adding a placeholder for the fire heat flux provided by smoke-dust code.
3. A change to the transpiration function to take into account the non-linear distribution of roots.

### Associated UFSWM Issue to close

Closes https://github.com/ufs-community/ccpp-physics/issues/117

### Subcomponent Pull Requests

[
](https://github.com/NOAA-GSL/fv3atm/pull/165)
[
](https://github.com/ufs-community/ccpp-physics/pull/116)### Blocking Dependencies

### Subcomponents involved:
- [ ] AQM
- [ ] CDEPS
- [ ] CICE
- [ ] CMEPS
- [ ] CMakeModules
- [x] FV3
- [ ] GOCART
- [ ] HYCOM
- [ ] MOM6
- [ ] NOAHMP
- [ ] WW3
- [ ] stochastic_physics
- [ ] none

### Input data
- [x] No changes are expected to input data.
- [ ] Changes are expected to input data:
  - [ ] New input data.
  - [ ] Updated input data.

### Regression Tests:
- [ ] No changes are expected to any regression test.
- [x] Changes are expected to the following tests:

<details><summary>Tests effected by changes in this PR:</summary>

</details>

### Libraries

- [ ] Not Needed
- [ ] Needed
  - [ ] Create separate issue in [JCSDA/spack-stack](https://github.com/JCSDA/spack-stack) asking for update to library. Include library name, library version.
  - [ ] Add issue link from JCSDA/spack-stack following this item 
RUC LSM has a modification that will change the results with the use of RUC LSM.

<details><summary>Code Managers Log</summary>

- [ ] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR.
- [ ] Move new/updated input data on RDHPCS Hera and propagate input data changes to all supported systems.
  - [ ] N/A

### Testing Log:
- RDHPCS
  - [x] Hera
  - [ ] Orion
  - [ ] Hercules
  - [ ] Jet
  - [ ] Gaea
  - [ ] Cheyenne
- WCOSS2
  - [ ] Dogwood/Cactus
  - [ ] Acorn
- CI
  - [ ] Completed
- opnReqTest
  - [ ] N/A
  - [ ] Log attached to comment
</details>

### CODE REVIEW

**Comment by tanyasmirnova** [comment]
@SamuelTrahanNOAA Sam, could you please review this PR?

**Comment by tanyasmirnova** [comment]
@SamuelTrahanNOAA Sam, I would like to add you as a reviewer.

**Comment by SamuelTrahanNOAA** [comment]
Your hera test logs say the tests are still running. You'll need to add the hera test log again and push it once the tests have completed.

**Comment by tanyasmirnova** [comment]
@SamuelTrahanNOAA Yes, I will update the log when the full regression test passes.

**Comment by grantfirl** [comment]
@tanyasmirnova I see that the RT log that you pushed shows success when testing against your own baseline, but shouldn't there be RT failures testing against the existing ufs-community baselines?

**Comment by grantfirl** [comment]
@tanyasmirnova @jkbk2004 @joeolson42 There is a desire to combine this PR with https://github.com/ufs-community/ufs-weather-model/pull/1945 to save time in the queue. @joeolson42 is going to run RTs on https://github.com/ufs-community/ufs-weather-model/pull/1945 once @AndersJensen-NOAA pushes some small changes to verify that the correct RTs are failing. Once we know that the RTs for this PR are failing as expected (if any), we should be OK to combine them. @tanyasmirnova I can re-run RTs for this one against the ufs-community baselines and report the list of failed RTs (if any) here.

**Comment by tanyasmirnova** [comment]
@grantfirl Grant, before making my own baseline I compared against the ufs-community baseline, and only suites containing RUC LSM failed. After that I created my own baseline. I would appreciate if you could rerun the RT against ufs-community baselines.

**Comment by grantfirl** [comment]
The following tests failed in check_result. @tanyasmirnova This seems like more tests than I expected, including some that don't use RUC LSM. Does this make sense to you?

003 cpld_control_gfsv17_iau_intel failed in check_result
cpld_control_gfsv17_iau_intel 003 failed in run_test
017 cpld_bmark_p8_intel failed in check_result
cpld_bmark_p8_intel 017 failed in run_test
030 control_flake_intel failed in check_result
control_flake_intel 030 failed in run_test
032 control_CubedSphereGrid_parallel_intel failed in check_result
control_CubedSphereGrid_parallel_intel 032 failed in run_test
033 control_latlon_intel failed in check_result
control_latlon_intel 033 failed in run_test
034 control_wrtGauss_netcdf_parallel_intel failed in check_result
control_wrtGauss_netcdf_parallel_intel 034 failed in run_test
036 control_c192_intel failed in check_result
control_c192_intel 036 failed in run_test
037 control_c384_intel failed in check_result
control_c384_intel 037 failed in run_test
038 control_c384gdas_intel failed in check_result
control_c384gdas_intel 038 failed in run_test
039 control_stochy_intel failed in check_result
control_stochy_intel 039 failed in run_test
041 control_lndp_intel failed in check_result
control_lndp_intel 041 failed in run_test
042 control_iovr4_intel failed in check_result
control_iovr4_intel 042 failed in run_test
043 control_iovr5_intel failed in check_result
control_iovr5_intel 043 failed in run_test
044 control_p8_intel failed in check_result
control_p8_intel 044 failed in run_test
045 control_p8_ugwpv1_intel failed in check_result
control_p8_ugwpv1_intel 045 failed in run_test
047 control_qr_p8_intel failed in check_result
control_qr_p8_intel 047 failed in run_test
049 control_decomp_p8_intel failed in check_result
control_decomp_p8_intel 049 failed in run_test
050 control_2threads_p8_intel failed in check_result
control_2threads_p8_intel 050 failed in run_test
051 control_p8_lndp_intel failed in check_result
control_p8_lndp_intel 051 failed in run_test
052 control_p8_rrtmgp_intel failed in check_result
control_p8_rrtmgp_intel 052 failed in run_test
053 control_p8_mynn_intel failed in check_result
control_p8_mynn_intel 053 failed in run_test
054 merra2_thompson_intel failed in check_result
merra2_thompson_intel 054 failed in run_test
055 regional_control_intel failed in check_result
regional_control_intel 055 failed in run_test
057 regional_control_qr_intel failed in check_result
regional_control_qr_intel 057 failed in run_test
059 regional_decomp_intel failed in check_result
regional_decomp_intel 059 failed in run_test
060 regional_2threads_intel failed in check_result
regional_2threads_intel 060 failed in run_test
063 regional_2dwrtdecomp_intel failed in check_result
regional_2dwrtdecomp_intel 063 failed in run_test
064 regional_wofs_intel failed in check_result
regional_wofs_intel 064 failed in run_test
065 rap_control_intel failed in check_result
rap_control_intel 065 failed in run_test
066 regional_spp_sppt_shum_skeb_intel failed in check_result
regional_spp_sppt_shum_skeb_intel 066 failed in run_test
067 rap_decomp_intel failed in check_result
rap_decomp_intel 067 failed in run_test
068 rap_2threads_intel failed in check_result
rap_2threads_intel 068 failed in run_test
070 rap_sfcdiff_intel failed in check_result
rap_sfcdiff_intel 070 failed in run_test
071 rap_sfcdiff_decomp_intel failed in check_result
rap_sfcdiff_decomp_intel 071 failed in run_test
073 hrrr_control_intel failed in check_result
hrrr_control_intel 073 failed in run_test
074 hrrr_control_qr_intel failed in check_result
hrrr_control_qr_intel 074 failed in run_test
075 hrrr_control_decomp_intel failed in check_result
hrrr_control_decomp_intel 075 failed in run_test
076 hrrr_control_2threads_intel failed in check_result
hrrr_control_2threads_intel 076 failed in run_test
079 rrfs_v1beta_intel failed in check_result
rrfs_v1beta_intel 079 failed in run_test
084 control_ras_intel failed in check_result
control_ras_intel 084 failed in run_test
086 control_p8_faster_intel failed in check_result
control_p8_faster_intel 086 failed in run_test
087 regional_control_faster_intel failed in check_result
regional_control_faster_intel 087 failed in run_test
098 rap_control_debug_intel failed in check_result
rap_control_debug_intel 098 failed in run_test
099 hrrr_control_debug_intel failed in check_result
hrrr_control_debug_intel 099 failed in run_test
100 hrrr_gf_debug_intel failed in check_result
hrrr_gf_debug_intel 100 failed in run_test
101 hrrr_c3_debug_intel failed in check_result
hrrr_c3_debug_intel 101 failed in run_test
102 rap_unified_drag_suite_debug_intel failed in check_result
rap_unified_drag_suite_debug_intel 102 failed in run_test
103 rap_diag_debug_intel failed in check_result
rap_diag_debug_intel 103 failed in run_test
104 rap_cires_ugwp_debug_intel failed in check_result
rap_cires_ugwp_debug_intel 104 failed in run_test
105 rap_unified_ugwp_debug_intel failed in check_result
rap_unified_ugwp_debug_intel 105 failed in run_test
106 rap_lndp_debug_intel failed in check_result
rap_lndp_debug_intel 106 failed in run_test
107 rap_progcld_thompson_debug_intel failed in check_result
rap_progcld_thompson_debug_intel 107 failed in run_test
109 rap_sfcdiff_debug_intel failed in check_result
rap_sfcdiff_debug_intel 109 failed in run_test
112 rap_clm_lake_debug_intel failed in check_result
rap_clm_lake_debug_intel 112 failed in run_test
113 rap_flake_debug_intel failed in check_result
rap_flake_debug_intel 113 failed in run_test
115 regional_spp_sppt_shum_skeb_dyn32_phy32_intel failed in check_result
regional_spp_sppt_shum_skeb_dyn32_phy32_intel 115 failed in run_test
116 rap_control_dyn32_phy32_intel failed in check_result
rap_control_dyn32_phy32_intel 116 failed in run_test
117 hrrr_control_dyn32_phy32_intel failed in check_result
hrrr_control_dyn32_phy32_intel 117 failed in run_test
118 hrrr_control_qr_dyn32_phy32_intel failed in check_result
hrrr_control_qr_dyn32_phy32_intel 118 failed in run_test
119 rap_2threads_dyn32_phy32_intel failed in check_result
rap_2threads_dyn32_phy32_intel 119 failed in run_test
120 hrrr_control_2threads_dyn32_phy32_intel failed in check_result
hrrr_control_2threads_dyn32_phy32_intel 120 failed in run_test
121 hrrr_control_decomp_dyn32_phy32_intel failed in check_result
hrrr_control_decomp_dyn32_phy32_intel 121 failed in run_test
125 conus13km_control_intel failed in check_result
conus13km_control_intel 125 failed in run_test
128 rap_control_dyn64_phy32_intel failed in check_result
rap_control_dyn64_phy32_intel 128 failed in run_test
129 rap_control_debug_dyn32_phy32_intel failed in check_result
rap_control_debug_dyn32_phy32_intel 129 failed in run_test
130 hrrr_control_debug_dyn32_phy32_intel failed in check_result
hrrr_control_debug_dyn32_phy32_intel 130 failed in run_test
131 conus13km_debug_intel failed in check_result
conus13km_debug_intel 131 failed in run_test
132 conus13km_debug_qr_intel failed in check_result
conus13km_debug_qr_intel 132 failed in run_test
133 conus13km_debug_2threads_intel failed in check_result
conus13km_debug_2threads_intel 133 failed in run_test
134 conus13km_radar_tten_debug_intel failed in check_result
conus13km_radar_tten_debug_intel 134 failed in run_test
135 rap_control_dyn64_phy32_debug_intel failed in check_result
rap_control_dyn64_phy32_debug_intel 135 failed in run_test
136 hafs_regional_atm_intel failed in check_result
hafs_regional_atm_intel 136 failed in run_test
146 hafs_global_multiple_4nests_atm_intel failed in check_result
hafs_global_multiple_4nests_atm_intel 146 failed in run_test
147 hafs_global_multiple_4nests_atm_qr_intel failed in check_result
hafs_global_multiple_4nests_atm_qr_intel 147 failed in run_test
148 hafs_regional_specified_moving_1nest_atm_intel failed in check_result
hafs_regional_specified_moving_1nest_atm_intel 148 failed in run_test
177 control_atmwav_intel failed in check_result
control_atmwav_intel 177 failed in run_test
178 atmaero_control_p8_intel failed in check_result
atmaero_control_p8_intel 178 failed in run_test
179 atmaero_control_p8_rad_intel failed in check_result
atmaero_control_p8_rad_intel 179 failed in run_test
180 atmaero_control_p8_rad_micro_intel failed in check_result
atmaero_control_p8_rad_micro_intel 180 failed in run_test
185 control_stochy_gnu failed in check_result
control_stochy_gnu 185 failed in run_test
186 control_ras_gnu failed in check_result
control_ras_gnu 186 failed in run_test
187 control_p8_gnu failed in check_result
control_p8_gnu 187 failed in run_test
188 control_p8_ugwpv1_gnu failed in check_result
control_p8_ugwpv1_gnu 188 failed in run_test
189 control_flake_gnu failed in check_result
control_flake_gnu 189 failed in run_test
190 rap_control_gnu failed in check_result
rap_control_gnu 190 failed in run_test
191 rap_decomp_gnu failed in check_result
rap_decomp_gnu 191 failed in run_test
192 rap_2threads_gnu failed in check_result
rap_2threads_gnu 192 failed in run_test
194 rap_sfcdiff_gnu failed in check_result
rap_sfcdiff_gnu 194 failed in run_test
195 rap_sfcdiff_decomp_gnu failed in check_result
rap_sfcdiff_decomp_gnu 195 failed in run_test
197 hrrr_control_gnu failed in check_result
hrrr_control_gnu 197 failed in run_test
198 hrrr_control_qr_gnu failed in check_result
hrrr_control_qr_gnu 198 failed in run_test
199 hrrr_control_2threads_gnu failed in check_result
hrrr_control_2threads_gnu 199 failed in run_test
200 hrrr_control_decomp_gnu failed in check_result
hrrr_control_decomp_gnu 200 failed in run_test
203 rrfs_v1beta_gnu failed in check_result
rrfs_v1beta_gnu 203 failed in run_test
206 rap_control_debug_gnu failed in check_result
rap_control_debug_gnu 206 failed in run_test
207 hrrr_control_debug_gnu failed in check_result
hrrr_control_debug_gnu 207 failed in run_test
208 hrrr_gf_debug_gnu failed in check_result
hrrr_gf_debug_gnu 208 failed in run_test
209 hrrr_c3_debug_gnu failed in check_result
hrrr_c3_debug_gnu 209 failed in run_test
210 rap_diag_debug_gnu failed in check_result
rap_diag_debug_gnu 210 failed in run_test
212 rap_progcld_thompson_debug_gnu failed in check_result
rap_progcld_thompson_debug_gnu 212 failed in run_test
217 rap_flake_debug_gnu failed in check_result
rap_flake_debug_gnu 217 failed in run_test
218 rap_clm_lake_debug_gnu failed in check_result
rap_clm_lake_debug_gnu 218 failed in run_test
220 rap_control_dyn32_phy32_gnu failed in check_result
rap_control_dyn32_phy32_gnu 220 failed in run_test
221 hrrr_control_dyn32_phy32_gnu failed in check_result
hrrr_control_dyn32_phy32_gnu 221 failed in run_test
222 hrrr_control_qr_dyn32_phy32_gnu failed in check_result
hrrr_control_qr_dyn32_phy32_gnu 222 failed in run_test
223 rap_2threads_dyn32_phy32_gnu failed in check_result
rap_2threads_dyn32_phy32_gnu 223 failed in run_test
224 hrrr_control_2threads_dyn32_phy32_gnu failed in check_result
hrrr_control_2threads_dyn32_phy32_gnu 224 failed in run_test
225 hrrr_control_decomp_dyn32_phy32_gnu failed in check_result
hrrr_control_decomp_dyn32_phy32_gnu 225 failed in run_test
229 conus13km_control_gnu failed in check_result
conus13km_control_gnu 229 failed in run_test
232 rap_control_dyn64_phy32_gnu failed in check_result
rap_control_dyn64_phy32_gnu 232 failed in run_test
233 rap_control_debug_dyn32_phy32_gnu failed in check_result
rap_control_debug_dyn32_phy32_gnu 233 failed in run_test
234 hrrr_control_debug_dyn32_phy32_gnu failed in check_result
hrrr_control_debug_dyn32_phy32_gnu 234 failed in run_test
235 conus13km_debug_gnu failed in check_result
conus13km_debug_gnu 235 failed in run_test
236 conus13km_debug_qr_gnu failed in check_result
conus13km_debug_qr_gnu 236 failed in run_test
237 conus13km_debug_2threads_gnu failed in check_result
conus13km_debug_2threads_gnu 237 failed in run_test
238 conus13km_radar_tten_debug_gnu failed in check_result
conus13km_radar_tten_debug_gnu 238 failed in run_test
239 rap_control_dyn64_phy32_debug_gnu failed in check_result
rap_control_dyn64_phy32_debug_gnu 239 failed in run_test

**Comment by grantfirl** [comment]
@tanyasmirnova @SamuelTrahanNOAA Do either of you know if the list of failed RTs above are all expected failures? I want to merge this with Joe's PR, but I can't until I understand whether these failures are expected given the code changes.

**Comment by tanyasmirnova** [comment]
@grantfirl @SamuelTrahanNOAA Grant and Sam, I expected all RAP and HRRR tests to fail, but I am not sure why, for example, cpld_control_gfsv17_iau_intel test did not pass. I am trying to run a regression test in my directory on hera to see if I can confirm the same failures of non_RUC SDFs. Sam, do you have time to look into this? Thank you!

**Comment by tanyasmirnova** [comment]
@grantfirl @SamuelTrahanNOAA To me the only suspect is GFS_diagnostics.F90. I have also merged to my PR and upstream/develp on October 18, maybe this could change the results?

**Comment by SamuelTrahanNOAA** [comment]
> @grantfirl @SamuelTrahanNOAA Grant and Sam, I expected all RAP and HRRR tests to fail, but I am not sure why, for example, cpld_control_gfsv17_iau_intel test did not pass. I am trying to run a regression test in my directory on hera to see if I can confirm the same failures of non_RUC SDFs. Sam, do you have time to look into this? Thank you!

You should check the diagnostic fields. I'm wondering if your changes to GFS_diagnostics.F90 has had unintended effects on other configurations' output.

I see nothing in the physics or GFS_typedefs that could affect non-ruc non-smoke configurations.

**Comment by grantfirl** [comment]
> @grantfirl @SamuelTrahanNOAA Grant and Sam, I expected all RAP and HRRR tests to fail, but I am not sure why, for example, cpld_control_gfsv17_iau_intel test did not pass. I am trying to run a regression test in my directory on hera to see if I can confirm the same failures of non_RUC SDFs. Sam, do you have time to look into this? Thank you!

Yes, this is why I was confused as well. FYI, I ran regression tests both using this branch as-is and after updating to the latest develop branch, and I got the exact same failures. I also ran regression tests only omitting the changes to GFS_diagnostics.F90 in the fv3atm PR and also got the same failures.

For example, for the cpld_control_gfsv17_iau_intel test, only the GFSPRS.GrbF12 file is not identical.

**Comment by grantfirl** [comment]
I will double-check that the baseline is OK by running one of the tests that we don't expect to fail (cpld_control_gfsv17_iau_intel) with the top of the develop branch.

**Comment by tanyasmirnova** [comment]
@grantfirl @SamuelTrahanNOAA Thank you, Sam, for checking. In my opinion the changes to GFS_diagnostics.F90 should stay. The only change I can do in GFS_diagnostics.F90 is not to output surface fire heat flux and ratio of burned area. I can put     'if(Model%add_fire_heat_flux) then' around that output, and the default of add_fire_heat_flux is .false.

**Comment by tanyasmirnova** [comment]
Also, this fire_heat_flux is happening only with smoke physics on.

**Comment by grantfirl** [comment]
> @grantfirl @SamuelTrahanNOAA Thank you, Sam, for checking. In my opinion the changes to GFS_diagnostics.F90 should stay. The only change I can do in GFS_diagnostics.F90 is not to output surface fire heat flux and ratio of burned area. I can put 'if(Model%add_fire_heat_flux) then' around that output, and the default of add_fire_heat_flux is .false.

I'm not suggesting removing the changes in GFS_diagnostics, I'm just wanting to understand whether these code changes in totality are doing what you expect without changing something that you didn't expect.

**Comment by grantfirl** [comment]
@tanyasmirnova @SamuelTrahanNOAA So far, it seems like the only files that are listed as "NOT OK" in the non-HRRR/RAP tests are the GFSPRS.GrbFXX files. I don't know what is being written there.

**Comment by SamuelTrahanNOAA** [comment]
> @tanyasmirnova @SamuelTrahanNOAA So far, it seems like the only files that are listed as "NOT OK" in the non-HRRR/RAP tests are the GFSPRS.GrbFXX files. I don't know what is being written there.

This probably means a field that is not in the diagnostic files changed.

**Comment by SamuelTrahanNOAA** [comment]
> I will double-check that the baseline is OK by running one of the tests that we don't expect to fail (cpld_control_gfsv17_iau_intel) with the top of the develop branch.

Please try merging the top of the authoritative branches. It's possible the UPP failure is unrelated to Tanya's changes. I've run into many UPP bugs where an out-of-bounds read causes results to change or a test to fail after changing FV3 in unrelated ways. The last PR that was merged had a new version of UPP. With luck, this will fix our problems.

**Comment by grantfirl** [comment]
> > I will double-check that the baseline is OK by running one of the tests that we don't expect to fail (cpld_control_gfsv17_iau_intel) with the top of the develop branch.
> 
> Please try merging the top of the authoritative branches. It's possible the UPP failure is unrelated to Tanya's changes. I've run into many UPP bugs where an out-of-bounds read causes results to change or a test to fail after changing FV3 in unrelated ways. The last PR that was merged had a new version of UPP. With luck, this will fix our problems.

Ya, I saw that UPP was just updated. The original RTs that I did were with updates to the latest develop at the time (last Friday or so).

I'm going to try 2 things. I'll run one of the unexpected failures with the top of develop against the stored baselines. If that passes, I'll run the same individual test with the latest develop code merged in. If that passes, I'll kick off another round of all tests with the latest develop code merged in.

**Comment by grantfirl** [comment]
@tanyasmirnova @SamuelTrahanNOAA A quick update -- I think that there is good news. One of the unexpected tests that had been failing is now passing with the latest develops merged in. I'm guessing that there was something amiss with upp that was fixed in the latest commit. I'm now running all tests and will post results tomorrow.

**Comment by SamuelTrahanNOAA** [comment]
> @tanyasmirnova @SamuelTrahanNOAA A quick update -- I think that there is good news. One of the unexpected tests that had been failing is now passing with the latest develops merged in. I'm guessing that there was something amiss with upp that was fixed in the latest commit. I'm now running all tests and will post results tomorrow.

The plan was to combine this with #1945. Will you be able to do that?

**Comment by grantfirl** [comment]
> > @tanyasmirnova @SamuelTrahanNOAA A quick update -- I think that there is good news. One of the unexpected tests that had been failing is now passing with the latest develops merged in. I'm guessing that there was something amiss with upp that was fixed in the latest commit. I'm now running all tests and will post results tomorrow.
> 
> The plan was to combine this with #1945. Will you be able to do that?

Yes. I didn't want to combine them until each was exhibiting the expected behavior individually, though.

**Comment by grantfirl** [comment]
OK, I think that we're in business. Here is the list of failed RTs after updating the code to the latest develops. These are all expected, I think. I'll go ahead and merge this into Joe's branch for final testing and merge.

063 rap_control_intel failed in check_result
rap_control_intel 063 failed in run_test
064 regional_spp_sppt_shum_skeb_intel failed in check_result
regional_spp_sppt_shum_skeb_intel 064 failed in run_test
065 rap_decomp_intel failed in check_result
rap_decomp_intel 065 failed in run_test
066 rap_2threads_intel failed in check_result
rap_2threads_intel 066 failed in run_test
068 rap_sfcdiff_intel failed in check_result
rap_sfcdiff_intel 068 failed in run_test
069 rap_sfcdiff_decomp_intel failed in check_result
rap_sfcdiff_decomp_intel 069 failed in run_test
071 hrrr_control_intel failed in check_result
hrrr_control_intel 071 failed in run_test
072 hrrr_control_decomp_intel failed in check_result
hrrr_control_decomp_intel 072 failed in run_test
073 hrrr_control_2threads_intel failed in check_result
hrrr_control_2threads_intel 073 failed in run_test
094 rap_control_debug_intel failed in check_result
rap_control_debug_intel 094 failed in run_test
095 hrrr_control_debug_intel failed in check_result
hrrr_control_debug_intel 095 failed in run_test
096 hrrr_gf_debug_intel failed in check_result
hrrr_gf_debug_intel 096 failed in run_test
097 hrrr_c3_debug_intel failed in check_result
hrrr_c3_debug_intel 097 failed in run_test
098 rap_unified_drag_suite_debug_intel failed in check_result
rap_unified_drag_suite_debug_intel 098 failed in run_test
099 rap_diag_debug_intel failed in check_result
rap_diag_debug_intel 099 failed in run_test
100 rap_cires_ugwp_debug_intel failed in check_result
rap_cires_ugwp_debug_intel 100 failed in run_test
101 rap_unified_ugwp_debug_intel failed in check_result
rap_unified_ugwp_debug_intel 101 failed in run_test
102 rap_lndp_debug_intel failed in check_result
rap_lndp_debug_intel 102 failed in run_test
103 rap_progcld_thompson_debug_intel failed in check_result
rap_progcld_thompson_debug_intel 103 failed in run_test
105 rap_sfcdiff_debug_intel failed in check_result
rap_sfcdiff_debug_intel 105 failed in run_test
108 rap_clm_lake_debug_intel failed in check_result
rap_clm_lake_debug_intel 108 failed in run_test
109 rap_flake_debug_intel failed in check_result
rap_flake_debug_intel 109 failed in run_test
111 regional_spp_sppt_shum_skeb_dyn32_phy32_intel failed in check_result
regional_spp_sppt_shum_skeb_dyn32_phy32_intel 111 failed in run_test
112 rap_control_dyn32_phy32_intel failed in check_result
rap_control_dyn32_phy32_intel 112 failed in run_test
113 hrrr_control_dyn32_phy32_intel failed in check_result
hrrr_control_dyn32_phy32_intel 113 failed in run_test
114 rap_2threads_dyn32_phy32_intel failed in check_result
rap_2threads_dyn32_phy32_intel 114 failed in run_test
115 hrrr_control_2threads_dyn32_phy32_intel failed in check_result
hrrr_control_2threads_dyn32_phy32_intel 115 failed in run_test
116 hrrr_control_decomp_dyn32_phy32_intel failed in check_result
hrrr_control_decomp_dyn32_phy32_intel 116 failed in run_test
119 conus13km_control_intel failed in check_result
conus13km_control_intel 119 failed in run_test
122 rap_control_dyn64_phy32_intel failed in check_result
rap_control_dyn64_phy32_intel 122 failed in run_test
123 rap_control_debug_dyn32_phy32_intel failed in check_result
rap_control_debug_dyn32_phy32_intel 123 failed in run_test
124 hrrr_control_debug_dyn32_phy32_intel failed in check_result
hrrr_control_debug_dyn32_phy32_intel 124 failed in run_test
125 conus13km_debug_intel failed in check_result
conus13km_debug_intel 125 failed in run_test
126 conus13km_debug_qr_intel failed in check_result
conus13km_debug_qr_intel 126 failed in run_test
127 conus13km_debug_2threads_intel failed in check_result
conus13km_debug_2threads_intel 127 failed in run_test
128 conus13km_radar_tten_debug_intel failed in check_result
conus13km_radar_tten_debug_intel 128 failed in run_test
129 rap_control_dyn64_phy32_debug_intel failed in check_result
rap_control_dyn64_phy32_debug_intel 129 failed in run_test
180 rap_control_gnu failed in check_result
rap_control_gnu 180 failed in run_test
181 rap_decomp_gnu failed in check_result
rap_decomp_gnu 181 failed in run_test
182 rap_2threads_gnu failed in check_result
rap_2threads_gnu 182 failed in run_test
184 rap_sfcdiff_gnu failed in check_result
rap_sfcdiff_gnu 184 failed in run_test
185 rap_sfcdiff_decomp_gnu failed in check_result
rap_sfcdiff_decomp_gnu 185 failed in run_test
187 hrrr_control_gnu failed in check_result
hrrr_control_gnu 187 failed in run_test
188 hrrr_control_noqr_gnu failed in check_result
hrrr_control_noqr_gnu 188 failed in run_test
189 hrrr_control_2threads_gnu failed in check_result
hrrr_control_2threads_gnu 189 failed in run_test
190 hrrr_control_decomp_gnu failed in check_result
hrrr_control_decomp_gnu 190 failed in run_test
196 rap_control_debug_gnu failed in check_result
rap_control_debug_gnu 196 failed in run_test
197 hrrr_control_debug_gnu failed in check_result
hrrr_control_debug_gnu 197 failed in run_test
198 hrrr_gf_debug_gnu failed in check_result
hrrr_gf_debug_gnu 198 failed in run_test
199 hrrr_c3_debug_gnu failed in check_result
hrrr_c3_debug_gnu 199 failed in run_test
200 rap_diag_debug_gnu failed in check_result
rap_diag_debug_gnu 200 failed in run_test
202 rap_progcld_thompson_debug_gnu failed in check_result
rap_progcld_thompson_debug_gnu 202 failed in run_test
207 rap_flake_debug_gnu failed in check_result
rap_flake_debug_gnu 207 failed in run_test
208 rap_clm_lake_debug_gnu failed in check_result
rap_clm_lake_debug_gnu 208 failed in run_test
210 rap_control_dyn32_phy32_gnu failed in check_result
rap_control_dyn32_phy32_gnu 210 failed in run_test
211 hrrr_control_dyn32_phy32_gnu failed in check_result
hrrr_control_dyn32_phy32_gnu 211 failed in run_test
212 rap_2threads_dyn32_phy32_gnu failed in check_result
rap_2threads_dyn32_phy32_gnu 212 failed in run_test
213 hrrr_control_2threads_dyn32_phy32_gnu failed in check_result
hrrr_control_2threads_dyn32_phy32_gnu 213 failed in run_test
214 hrrr_control_decomp_dyn32_phy32_gnu failed in check_result
hrrr_control_decomp_dyn32_phy32_gnu 214 failed in run_test
217 conus13km_control_gnu failed in check_result
conus13km_control_gnu 217 failed in run_test
220 rap_control_dyn64_phy32_gnu failed in check_result
rap_control_dyn64_phy32_gnu 220 failed in run_test
221 rap_control_debug_dyn32_phy32_gnu failed in check_result
rap_control_debug_dyn32_phy32_gnu 221 failed in run_test
222 hrrr_control_debug_dyn32_phy32_gnu failed in check_result
hrrr_control_debug_dyn32_phy32_gnu 222 failed in run_test
223 conus13km_debug_gnu failed in check_result
conus13km_debug_gnu 223 failed in run_test
224 conus13km_debug_qr_gnu failed in check_result
conus13km_debug_qr_gnu 224 failed in run_test
225 conus13km_debug_2threads_gnu failed in check_result
conus13km_debug_2threads_gnu 225 failed in run_test
226 conus13km_radar_tten_debug_gnu failed in check_result
conus13km_radar_tten_debug_gnu 226 failed in run_test
227 rap_control_dyn64_phy32_debug_gnu failed in check_result
rap_control_dyn64_phy32_debug_gnu 227 failed in run_test

**Comment by grantfirl** [comment]
[RegressionTests_hera.log](https://github.com/ufs-community/ufs-weather-model/files/13367058/RegressionTests_hera.log)

**Comment by DeniseWorthen** [comment]
@grantfirl This is showing as having conflicting FV3 hashes. Do we know for sure the test was w/ an up-to-date FV3?

**Comment by grantfirl** [comment]
> @grantfirl This is showing as having conflicting FV3 hashes. Do we know for sure the test was w/ an up-to-date FV3?

Yes. I manually updated all branches on Hera before kicking off the tests. I didn't push the updates to Tanya's branch because I don't think that I have permissions, and I'm merging this into #1945 right now anyway.

**Comment by tanyasmirnova** [comment]
@grantfirl Thank you for you work and good news.

**Comment by grantfirl** [comment]
This PR has been combined with https://github.com/ufs-community/ufs-weather-model/pull/1945.

File: tests/comp.log [hunk 1]
```diff
@@ -0,0 +1,1807 @@
++ SECONDS=0
```
Comment (DeniseWorthen) [inline tests/comp.log:1]: This file should not be committed.

File: tests/modules.fv3.lua [hunk 2]
```diff
@@ -0,0 +1,27 @@
+help([[
```
Comment (DeniseWorthen) [inline tests/modules.fv3.lua:1]: This file should not be committed.

File: tests/rt.conf.small [hunk 3]
```diff
@@ -0,0 +1,229 @@
+### RT.CONF FORMATTING ###
```
Comment (DeniseWorthen) [inline tests/rt.conf.small:1]: This file should not be committed

File: tests/rt.sh [hunk 4]
```diff
@@ -352,8 +352,11 @@ elif [[ $MACHINE_ID = hera ]]; then
   PARTITION=
   dprefix=/scratch1/NCEPDEV
   DISKNM=/scratch2/NAGAPE/epic/UFS-WM_RT
-  STMP=$dprefix/stmp4
-  PTMP=$dprefix/stmp2
+  #STMP=$dprefix/stmp4
```
Comment (DeniseWorthen) [inline tests/rt.sh:355]: These need to be reverted to the standard locations.

**Comment by grantfirl** [comment]
@DeniseWorthen All of the extra files and edited rt.sh were removed when I merged this into #1945. I was assuming that this PR would be closed as soon as #1945 is merged.

**Comment by DeniseWorthen** [comment]
@grantfirl Thanks. I wasn't sure which PR was the receiving PR for both sets of changes.

**Comment by jkbk2004** [comment]
Merged with #1945

**Pull request closed without merging**